pygmt/helpers/decorators.py

```diff
@@ -204,9 +204,9 @@ def fmt_docstring(module_func):
         filler_text["aliases"] = "\n".join(aliases)
 
     for marker, text in COMMON_OPTIONS.items():
-        # Remove the indentation from the multiline strings so that it doesn't
-        # mess up the original docstring
-        filler_text[marker] = textwrap.dedent(text)
+        # Remove the indentation and the first line break from the multiline
+        # strings so that it doesn't mess up the original docstring
+        filler_text[marker] = textwrap.dedent(text.lstrip("\n"))
```

Comment:
`lstrip("\n")` is the trick to use raw strings while keeping the readability of the codes.

Reply:
Nice! This does look much cleaner!